v4/export/sql.go

```diff
@@ -50,7 +50,9 @@ func ShowCreateDatabase(db *sql.Conn, database string) (string, error) {
 	query := fmt.Sprintf("SHOW CREATE DATABASE `%s`", escapeString(database))
 	err := simpleQuery(db, query, handleOneRow)
 	if err != nil {
-		return "", errors.Annotatef(err, "sql: %s", query)
```

Comment:
Might be good to keep this as log entry, but didn't see a simple way to do that.